### DISCUSSION TITLE
panic: Unable to parse database: yaml: line 1: did not find expected key

### DISCUSSION THREAD

**Discussion by lStan464l** [authelia/authelia#4179] [Q&A] [answered]
Unsure why I am getting this.

Sanity checked my Config over the last 3 days.

**panic: Unable to parse database: yaml: line 1: did not find expected key

goroutine 1 [running]:
github.com/authelia/authelia/v4/internal/authentication.NewFileUserProvider(0xc000274c48)
        github.com/authelia/authelia/v4/internal/authentication/file_user_provider.go:53 +0x137
github.com/authelia/authelia/v4/internal/commands.getProviders()
        github.com/authelia/authelia/v4/internal/commands/helpers.go:48 +0x1be
github.com/authelia/authelia/v4/internal/commands.cmdRootRun(0xc0001cf400?, {0xc00031f1c0?, 0x2?, 0x2?})
        github.com/authelia/authelia/v4/internal/commands/root.go:69 +0x258
github.com/spf13/cobra.(*Command).execute(0xc0001cf400, {0xc0000360a0, 0x2, 0x2})
        github.com/spf13/cobra@v1.5.0/command.go:876 +0x67b
github.com/spf13/cobra.(*Command).ExecuteC(0xc0001cf400)
        github.com/spf13/cobra@v1.5.0/command.go:990 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.5.0/command.go:918
main.main()
        github.com/authelia/authelia/v4/cmd/authelia/main.go:10 +0x1e**

**Comment by james-d-elliott** [comment, marked as answer]
Use valid YAML. Validation can be done using this information: https://www.authelia.com/configuration/methods/files/#yaml-validation

Example of a valid user DB YAML file can be found here: https://www.authelia.com/reference/guides/passwords/#yaml-format

**Reply by lStan464l** [reply]
Resolved. Appreciate the fast reply.

Any Donation links, once in sorted setting up. Would love too donate on Rolling monthly basis. 

Thanks Kindly.
